samples/azure-quantum/variational-quantum-eigensolver/README.md

```diff
@@ -0,0 +1,20 @@
+---
+page_type: sample
+author: guenp
+description: Variational Quantum Eigensolver
+ms.author: guenp@microsoft.com
+ms.date: 05/02/2022
+languages:
+- python
+products:
+- azure-quantum
+---
+
+# Variational Quantum Eigensolver
+
+This is an Azure Quantum sample notebook that illustrates how to implement and run a Variational Quantum Eigensolver (VQE) program.
+
+## Manifest
+
+- [VQE-qiskit-hydrogen-ionq-sim.ipynb](./VQE-qiskit-hydrogen-ionq-sim.ipynb): Azure Quantum notebook for running the sample on the IonQ simulator
+- [VQE-qiskit-hydrogen-quantinuum-emulator.ipynb](./VQE-qiskit-hydrogen-quantinuum-emulator.ipynb): Azure Quantum notebook for running the sample on the Quantinuum emulator
```

Comment:
This should be a bit more descriptive, and should provide a bit better context for readers. Thanks!

```suggestion
# Estimating the ground state energy of hydrogen using variational quantum eigensolvers (VQE) on Azure Quantum

This sample shows how to estimate the ground state energy of hydrogen using the Azure Quantum service. In particular, this sample uses the implementation of the variational quantum eigensolver algorithm provided with Qiskit to estimate minimum energies. The sample demonstrates running this VQE implementation the simulator provided by IonQ and on the emulator provided by Quantinuum.

## Manifest

- [VQE-qiskit-hydrogen-ionq-sim.ipynb](./VQE-qiskit-hydrogen-ionq-sim.ipynb): Python + Qiskit notebook demonstrating using VQE on the IonQ simulator.
- [VQE-qiskit-hydrogen-quantinuum-emulator.ipynb](./VQE-qiskit-hydrogen-quantinuum-emulator.ipynb): Python + Qiskit notebook demonstrating using VQE on the Quantinuum emulator.

## See Also

To learn more about variational quantum eigensolvers, see the introduction at https://docs.microsoft.com/samples/microsoft/quantum/variational-quantum-algorithms/.
```